sql/connect/common/src/test/resources/query-tests/explain-results/from_protobuf_messageClassName_descFilePath.explain

```diff
@@ -1,2 +1,2 @@
-Project [from_protobuf(bytes#0, StorageLevel, Some([B)) AS from_protobuf(bytes)#0]
+Project [from_protobuf(bytes#0, StorageLevel, Some([B)) AS from_protobuf(bytes, StorageLevel, X'0AFC010A0C636F6D6D6F6E2E70726F746F120D737061726B2E636F6E6E65637422B0010A0C53746F726167654C6576656C12190A087573655F6469736B18012001280852077573654469736B121D0A0A7573655F6D656D6F727918022001280852097573654D656D6F727912200A0C7573655F6F66665F68656170180320012808520A7573654F66664865617012220A0C646573657269616C697A6564180420012808520C646573657269616C697A656412200A0B7265706C69636174696F6E180520012805520B7265706C69636174696F6E42220A1E6F72672E6170616368652E737061726B2E636F6E6E6563742E70726F746F5001620670726F746F33', NULL)#0]
```

Comment:
This part, I'm not sure why even plan is updated. I roughly suspect that this is maybe related to the fact that we switch the Protobuf functions into internal expression?

Reply:
Let's fix. I don't think we should show binaries here

Reply:
Hm.. I had some investigation and I think this is natural behavior because `from_avro` and `to_avro` also shows same behavior as below when we decouple them from `SparkConnectPlanner`:

```diff
- Project [from_avro(bytes#0, {"type": "int", "name": "id"}, (mode,FAILFAST), (compression,zstandard)) AS from_avro(bytes)#0]
+ Project [from_avro(bytes#0, {"type": "int", "name": "id"}, (mode,FAILFAST), (compression,zstandard)) AS from_avro(bytes, {"type": "int", "name": "id"}, map(mode, FAILFAST, compression, zstandard))#0]
``` 

The reason we show binary here is that the third argument of `from_protobuf` can actually accept data in binary format, so it just shows the input parameter itself as is (instead of some sort of “binaryized” data) so I guess it is fine??

Reply:
The alias is showing a binary, not the actual expression. We could update the plan we generate and add a proto, alternatively we can fix the pretty printing (or whatever we use for alias generation).